README.md

```diff
@@ -77,6 +77,8 @@ corresponding to the name of the specific module source subclass.
 
 For JavaScript modules, the module source phase is then specified to return
 a `ModuleSource` object, representing an ECMAScript Module Source.
+The `ModuleSource.prototype` is a `%ModuleSourcePrototype%`.
```

Comment:
nit: p sure neither of these should have articles
```suggestion
`ModuleSource.prototype` is `%ModuleSourcePrototype%`.
```

Reply:
I think I already covered this.